lib/control/control-commands.c

```diff
@@ -134,13 +134,30 @@ control_connection_config(GString *command, gpointer user_data)
       goto exit;
     }
 
+  if (g_str_equal(arguments[1], "VERIFY"))
+    {
+      // verify = compare active config against config file
+      // verify_config();
```

Comment:
I would rather update the `result` with not yet implemented, or other text rather than commented out code.